src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
 
 export const Navbar: React.FunctionComponent = () => {
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="emptySpacingBox"></div>
+
+    <div id="menuToggle">
+      <input type="checkbox" />
```

Comment:
This is very clever! You're using [the checkbox hack](https://css-tricks.com/the-checkbox-hack/), which is some seriously advanced stuff. Well done!

However, from that article:
![image](https://user-images.githubusercontent.com/4410524/188863263-efd69ad4-66ae-4b11-ae9d-0031503de5ab.png)

"Questionable semantics" is right - this solution:
1. is fairly brittle - it would only take a small bug/difference in CSS handling to totally wreck it
2. isn't semantic or accessible - a screen reader or other HTML-focused viewers would have hard times figuring out what was going on with this

Functionality/interactivity should (as the article suggests) be implemented using Javascript. React makes this very easy - you can store the boolean "is the menu open" variable in the _state_ of the navbar component, and have a class appear conditionally on your elements:
```
<nav className={`menu ${isMenuOpen ? "expanded" : "collapsed"}`}>
```
Then use a `<button>` (it is probably semantically the element that makes most sense here) to toggle that state!

So, in summary - very clever solution indeed, but I'm sorry to say not appropriate for an accessible website!

Reply:
Done